package.json

```diff
@@ -8,7 +8,7 @@
   },
   "dependencies": {
     "@adobe/gatsby-theme-aio": "4.8.3",
-    "gatsby": "4.25.5",
+    "gatsby": "4.22.0",
```

Comment:
Was there a reason for downgrading gatsby?

Reply:
@jeff-matthews: Yes. The `aio-theme` does not support gatsby versions beyond 4.22.0. This was an issue I brought up last year, but was promptly closed for some reason. See https://jira.corp.adobe.com/browse/DEVSITE-532.

Reply:
Thanks @bdenham. I totally forgot about this.